Rubberduck.SmartIndenter/IndenterSettings.cs

```diff
@@ -134,11 +134,13 @@ public IndenterSettings(bool skipRegistry)
             {
                 try
                 {
-                    var reg = Registry.CurrentUser.OpenSubKey(@"Software\Microsoft\VBA\6.0\Common", false) ??
-                              Registry.CurrentUser.OpenSubKey(@"Software\Microsoft\VBA\7.0\Common", false);
-                    if (reg != null)
+                    using (var reg = Registry.CurrentUser.OpenSubKey(@"Software\Microsoft\VBA\6.0\Common", false) ??
+                                     Registry.CurrentUser.OpenSubKey(@"Software\Microsoft\VBA\7.0\Common", false))
```

Comment:
`OpenSubKey` can return `null`. This would probably be better without a `using` block here, since there is already a `reg != null` test below where it can be explicitly `Dispose`'d